<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="SSO"
           tests="3"
           failures="3"
           errors="0"
           skipped="0"
           time="0.516">
  <properties />
            <testcase name="videoAjaxMarkNomark" time="0.172" classname="com.ezijing.qa.datacenter.video.GetVideoAjaxMark">
                <failure type="java.lang.AssertionError"
                  message="expected:&lt;404&gt; but was:&lt;400&gt;"
              >
<![CDATA[java.lang.AssertionError: expected:<404> but was:<400>
  org.junit.Assert.fail(Assert.java:88)
  org.junit.Assert.failNotEquals(Assert.java:743)
  org.junit.Assert.assertEquals(Assert.java:118)
  org.junit.Assert.assertEquals(Assert.java:555)
  org.junit.Assert.assertEquals(Assert.java:542)
  com.ezijing.qa.datacenter.video.GetVideoAjaxMark.videoAjaxMarkNomark(GetVideoAjaxMark.java:73)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:46)
  org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:37)
  java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>

          </testcase>
            <testcase name="videoAjaxMarkNormal" time="0.172" classname="com.ezijing.qa.datacenter.video.GetVideoAjaxMark">
                <failure type="java.lang.AssertionError"
                  message="expected:&lt;404&gt; but was:&lt;200&gt;"
              >
<![CDATA[java.lang.AssertionError: expected:<404> but was:<200>
  org.junit.Assert.fail(Assert.java:88)
  org.junit.Assert.failNotEquals(Assert.java:743)
  org.junit.Assert.assertEquals(Assert.java:118)
  org.junit.Assert.assertEquals(Assert.java:555)
  org.junit.Assert.assertEquals(Assert.java:542)
  com.ezijing.qa.datacenter.video.GetVideoAjaxMark.videoAjaxMarkNormal(GetVideoAjaxMark.java:44)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:46)
  org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:37)
  java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>

          </testcase>
            <testcase name="videoAjaxMarkNoVideoid" time="0.172" classname="com.ezijing.qa.datacenter.video.GetVideoAjaxMark">
                <failure type="java.lang.AssertionError"
                  message="expected:&lt;404&gt; but was:&lt;400&gt;"
              >
<![CDATA[java.lang.AssertionError: expected:<404> but was:<400>
  org.junit.Assert.fail(Assert.java:88)
  org.junit.Assert.failNotEquals(Assert.java:743)
  org.junit.Assert.assertEquals(Assert.java:118)
  org.junit.Assert.assertEquals(Assert.java:555)
  org.junit.Assert.assertEquals(Assert.java:542)
  com.ezijing.qa.datacenter.video.GetVideoAjaxMark.videoAjaxMarkNoVideoid(GetVideoAjaxMark.java:58)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:46)
  org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:37)
  java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
